tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/constant/ListConstantNegativeTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Let's fix the other few places as well.

Reply:
Addressed here [d97530c](https://github.com/ballerina-platform/ballerina-lang/pull/40621/commits/d97530cb3d589035d6d66759b831507e9b0852f7)

Reply:
Better to review this https://github.com/ballerina-platform/ballerina-lang/pull/40621 before this